src/hyperiond/hyperiond.cpp

```diff
@@ -72,9 +72,9 @@ HyperionDaemon::HyperionDaemon(QString configFile, QObject *parent)
 		WarningIf(_qconfig.contains("logger"), Logger::getInstance("LOGGER"), "Logger settings overriden by command line argument");
```

Comment:
we should make a text typo hunt ;-) I'm sure we have more. Perhaps a english native can help ;-)